aptos-move/aptos-vm-logging/src/log_schema.rs

```diff
@@ -26,47 +25,42 @@ pub struct AdapterLogSchema {
 
     // transaction position in the list of transactions in the block,
     // 0 if the transaction is not part of a block (i.e. validation).
-    txn_id: usize,
+    pub(crate) txn_idx: usize,
```

Comment:
this is ugly, I'd just expose a getter fn

Reply:
oh I remember, txn_idx was taken by Schema which did smt else.. (it's bad name there). Called get_txn_idx for now.